atomic_scanners/pipeline-scanner/scanner.py

```diff
@@ -69,7 +69,16 @@ def __init__(self, path):
 
     def scan_release(self):
         env_vars_dict = dict()
-        with open(os.path.join(self.in_path, "etc/os-release")) as f:
+        etc_os_release = os.path.join(self.in_path, "etc/os-release")
+
+        if not os.path.isfile(etc_os_release):
+            self.json_out["Scan Results"]["OS Release"] = \
+                "Could not find OS release of image under test " \
+                "as /etc/os-release file does not exist."
+            return
+        # check if at all /etc/os-release exist even before opening it
```

Comment:
I believe the comment should be before we do the actual task. So, before the line:
```python
if not os.path.isfile(etc_os_release):
```
would make more sense.